### PR TITLE
Allow for more than one jurisdiction

### PR DESCRIPTION
**Pull request by evz**
On top of being able to import bill data for more than one organization, it also makes sense to allow for data to come from more than one jurisdiction. This change modifies the `OCD_JURISDICTION_ID` setting to `OCD_JURISDICTION_IDS` which expects a list of jurisdiction IDs that you'd like to import data from.

### CODE REVIEW

**Comment by evz** [comment]
@reginafcompton Any chance you can review this? It adds support for more than one jurisdiction and adds a relationship between Organizations and the Jurisdiction they are under. I'm pretty sure this shouldn't break anything in the currently running Councilmatics but we might want to cut a version before we merge this in so that we don't accidentally do that.

File: councilmatic_core/management/commands/import_data.py [hunk 1]
```diff
@@ -40,10 +40,8 @@
 
 session = requests.Session()
 
-for configuration in ['OCD_JURISDICTION_ID',
-                      'HEADSHOT_PATH',
-                      'LEGISLATIVE_SESSIONS'
-                      ]:
+for configuration in ['OCD_JURISDICTION_IDS',
+                      'HEADSHOT_PATH',]:
```
Comment (reginafcompton) [inline councilmatic_core/management/commands/import_data.py]: Why did you remove LEGISLATIVE_SESSIONS?
